data-es5.js

```diff
@@ -1606,6 +1606,7 @@ exports.tests = [
       edge13: true,
       firefox2: true,
       safari3_1: false,
+      safaritp: true,
```

Comment:
wow, this is an old one

Reply:
yes, it's great to see that JSC team started to focus on compatibility recently!